hera_cal/lst_stack/config.py

```diff
@@ -231,7 +274,7 @@ def _subsorted_list(x: Sequence[Sequence[Any]]) -> list[list[Any]]:
 
 @attrs.define(slots=False, frozen=True)
 class LSTBinConfigurator:
-    """
+    r"""
```

Comment:
Typo?

Reply:
Yeah. Although actually prepending an r to a string makes it "raw" so that it doesn't turn e.g. \n into newlines. I often have docstrings like this, but I think actually this is really a typo...

Reply:
Ah, no it wasn't a typo. It is mean to be a literal string -- there's some regex in there.